renovate.json5

```
@@ -0,0 +1,22 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": [
+    "config:base"
+  ],
+  "packageRules": [
+    {
+      // navigation-fragment 2.7.0 and above require android api 34+, which we are not ready for
+      // yet due to android gradle plugin only supporting min 33.
+      "matchPackagePrefixes": ["androidx.navigation"],
+      "matchCurrentVersion": "2.6.0",
```

this will still give you patch releases (should they be made)
```suggestion
      "matchUpdateTypes": ["major", "minor"],
```

---

Thanks, definitely didn't think about patch releases. New at this and cribbing from your prior work! :)